src/wazuh_modules/vulnerability_scanner/tests/unit/scanContext_test.cpp

```diff
@@ -763,7 +760,8 @@ TEST_F(ScanContextTest, TestSyscollectorDeltasNoOperation)
 {
     flatbuffers::Parser parser;
     ASSERT_TRUE(parser.Parse(syscollector_deltas_SCHEMA));
-    ASSERT_FALSE(parser.Parse(DELTA_NO_OPERATION_MSG.c_str()));
+    // Parser does not fail when operation is missing.
```

Comment:
We can ask about this.

Reply:
Yeap, I fixed the tests, but I'm not sure why that fbs changed.

Reply:
@Dwordcito 

We think that the `required` flags in the .fbs help to discard invalid messages. But this has changed.